roles/system/fedora-workstation/tasks/services.yml

```diff
@@ -4,6 +4,9 @@
     name: docker
     state: started
     enabled: yes
+  # On Fedora 31 or later, Docker is broken with CgroupsV2.
+  # https://fedoraproject.org/wiki/Changes/CGroupsV2
+  ignore_errors: yes
```

Comment:
Let's add optional deployment logic to fall back to cgroups v1, using grub kernel option
as documented on:
- https://fedoraproject.org/wiki/Changes/CGroupsV2#Upgrade.2Fcompatibility_impact
- https://medium.com/@drpdishant/installing-docker-on-fedora-31-beta-6c7a05901613

Reply:
@lhoss Thanks for the suggestion. In this case, I want to play around with cgroups V2 with Podman, so I I will wait for Docker to eventually build support in for cgroups V2. Once this is done, I'll remove this `ignore_errors` line. I appreciate the pointers though!

Reply:
For what it's worth, this was reverted in #91, as the `moby-engine` in Fedora now supports cgroups v2 :slightly_smiling_face: